apps/client/src/common/utils/socket.ts

```diff
@@ -139,6 +139,12 @@ export const connectSocket = () => {
           updateDevTools(serverPayload);
           break;
         }
+        case 'ontime-patch': {
+          patchRuntime(payload);
```

Comment:
question: what can we expect of the patch? is it a deep patch or are all the objects replaced at top level?

Reply:
that would be our decision, but right now it is only one key deep